example/rest-api/art-institute/art-institute.php

```diff
@@ -167,6 +206,10 @@ function register_aic_block(): void {
 				'query' => $search_art_query,
 				'type' => 'search',
 			],
+			[
+				'query' => $collection_query, 
+				'type' => 'collection',
```

Comment:
Makes sense to me. Ultimately I'd love to get rid of this `type` attribute and let it be inferred from the input schema, but works for now!